### PR TITLE
 Change the location of 'allocatable'

### PR DESCRIPTION
**Pull request by chi-xu**
Change the location of 'allocatable'

### CODE REVIEW

**Comment by chi-xu** [comment]
I signed

**Comment by chi-xu** [comment]
i signed

File: pkg/api/v1/resource/helpers.go [hunk 1]
```diff
@@ -107,7 +107,7 @@ func ExtractResourceValueByContainerName(fs *v1.ResourceFieldSelector, pod *v1.P
 }
 
 // ExtractResourceValueByContainerNameAndNodeAllocatable extracts the value of a resource
-// by providing container name and node allocatable
+// by providing allocatable container name and node
```
Comment (lavalamp) [inline pkg/api/v1/resource/helpers.go]: "node allocatable" would be very bad english if it didn't refer to a kubernetes specific feature. https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable

**Pull request closed without merging**